stats.js

```diff
@@ -107,19 +97,120 @@ const getTestingData = async () => {
     delete aggregateByDate[date].county
     delete aggregateByDate[date].test_date
   }
+  const dates = Object.keys(aggregateByDate).sort((a, b) => {
+    return Date.parse(a) > Date.parse(b) ? 1 : -1
+  })
+  // Calculate the moving averages for the aggregate dates.
+  dates.forEach((date, idx) => {
+    // TODO: move me into a separate function.
+    const mvgAvgDays = Math.min(idx + 1, movingAvgDays)
+    let newPositivesSum = 0
+    let totalTestsSum = 0
+    for (let i = 0; i < mvgAvgDays; i++) {
+      const curIdx = idx - i
+      newPositivesSum += aggregateByDate[dates[curIdx]].new_positives
+      totalTestsSum += aggregateByDate[dates[curIdx]].total_number_of_tests
+    }
+    aggregateByDate[dates[idx]].average_number_of_tests = parseInt(
+      totalTestsSum / mvgAvgDays
+    )
+    aggregateByDate[dates[idx]].average_new_positives = parseInt(
+      newPositivesSum / mvgAvgDays
+    )
+  })
   for (const county in byCounty) {
+    byCounty[county].sort((a, b) => {
+      return Date.parse(a.test_date) > Date.parse(b.test_date) ? 1 : -1
+    })
+    // Calculate moving averages by county.
+    byCounty[county].forEach((record, idx) => {
+      // TODO: move me into a separate function.
+      const mvgAvgDays = Math.min(idx + 1, movingAvgDays)
+      let newPositivesSum = 0
+      let totalTestsSum = 0
+      for (let i = 0; i < mvgAvgDays; i++) {
+        const curIdx = idx - i
+        newPositivesSum += byCounty[county][curIdx].new_positives
+        totalTestsSum += byCounty[county][curIdx].total_number_of_tests
+      }
+      record.average_number_of_tests = parseInt(totalTestsSum / mvgAvgDays)
+      record.average_new_positives = parseInt(newPositivesSum / mvgAvgDays)
```

Comment:
Can we please have at least 2 decimal places in the averages, not integers - some of the county testing positives are around `0`, `1`, `2` etc